src/sentry/static/sentry/app/components/alertLink.tsx

```diff
@@ -33,10 +33,10 @@ export default class AlertLink extends React.Component<Props> {
     return (
       <StyledLink
         to={(this.props as PropsWithTo).to}
-        href={(this.props as PropsWithHref).href}
         onClick={onClick}
         size={size}
         priority={priority}
+        external
```

Comment:
Are alert links always external?

Reply:
Well, they were using the `ExternalLink` component before

Reply:
Good point. Didn't think of that.